src/puppetlabs/puppetdb/queue.clj

```diff
@@ -400,36 +400,52 @@
    ;; to get rid of it somehow when we refactor mq-listener and command.clj.
    (SortedCommandBuffer. (TreeMap.) (HashMap.) n delete-update-fn)))
 
+(defn make-q-cmd-map
```

Comment:
rename this func and update doc string, these maps are passed along the channel now and not put into the atom directly